Networking/Networking/ApplicationPassword/ApplicationPasswordUseCase.swift

```diff
@@ -28,3 +34,219 @@ protocol ApplicationPasswordUseCase {
     ///
     func deletePassword() async throws
 }
+
+final class DefaultApplicationPasswordUseCase: ApplicationPasswordUseCase {
+    /// WordPress.com Credentials.
+    ///
+    private let credentials: Credentials
+
+    /// SiteID needed when using WPCOM credentials
+    ///
+    private let siteID: Int64
+
+    /// To generate and delete application password
+    ///
+    private let network: Network
+
+    /// Stores the application password
+    ///
+    private let keychain: Keychain
+
+    /// Used to name the password in wpadmin.
+    ///
+    private var applicationPasswordName: String {
+        get async {
+            await UIDevice.current.model
```

Comment:
According to the internal discussion [p1669977432588639/1669965496.085079-slack-C046HDZL87J], we should name the application in the following format: `<bundle-id>.ios-app-client.<model-name>`.